storage-proofs-porep/src/stacked/vanilla/proof.rs

```diff
@@ -1422,11 +1432,13 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
             let max_gpu_tree_batch_size = SETTINGS.max_gpu_tree_batch_size as usize;
 
             let _gpu_lock = GPU_LOCK.lock().expect("failed to get gpu lock");
+            let batcher = Some(
+                Batcher::pick_gpu(max_gpu_tree_batch_size).expect("failed to create gpu batcher"),
+            );
             let mut tree_builder = TreeBuilder::<Tree::Arity>::new(
                 #[cfg(feature = "gpu")]
-                Some(BatcherType::OpenCL),
+                batcher,
```

Comment:
The config flag in the line above is not needed as the whole function is only enabled when the `gpu` feature is enabled.

Reply:
Agreed, I noticed it was there, so preserved it for now (i.e. left alone).  Can remove before merging.